hack/kcp-edge.sh

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env bash
+# Copyright 2023 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: $0 (start or stop | --user dev or kit | -v
```

Comment:
I think we should consistently use the terms "user" and "contributor" to identify the two roles in question here.

I think we could use "user" and "developer" instead if the first mention of "developer" in the few most important docs is accompanied by an explicit indication that we mean developer _of this repo_.

Reply:
Fixed - using the local copies for the yaml manifests now, so the script don't need roles anymore